cosign/toolchain.bzl

```diff
@@ -0,0 +1,78 @@
+"""This module implements the cosign-specific toolchain rule."""
+
+def _to_manifest_path(ctx, file):
+    if file.short_path.startswith("../"):
+        return "external/" + file.short_path[3:]
```

Comment:
ohhh. this is going to depend on legacy_external_runfiles and will break in repositories that have `--nolegacy_external_runfiles`.

the correct approach is what we did with yq and just pass the binary itself to the toolchain info:
https://github.com/aspect-build/bazel-lib/blob/2689a48940007588e87f63acda704d72f2e2b6c2/lib/private/yq_toolchain.bzl#L111
and not do any conversion to manifest path